go.mod

```diff
@@ -2,24 +2,33 @@ module github.com/openshift/ptp-operator
 
 go 1.18
 
+replace k8s.io/client-go => k8s.io/client-go v0.25.2
+
+replace github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20220831193253-4950ae70c8ea
+
+replace github.com/openshift/api => github.com/openshift/api v0.0.0-20220831183848-09c070622e2c
+
+replace github.com/openshift/library-go => github.com/openshift/library-go v0.0.0-20220922140741-7772048e4447
+
 require (
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/go-logr/logr v1.2.3
 	github.com/golang/glog v1.0.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.2
-	github.com/openshift/api v0.0.0-20220831183848-09c070622e2c
-	github.com/openshift/client-go v0.0.0-20220831193253-4950ae70c8ea
-	github.com/openshift/library-go v0.0.0-20220922140741-7772048e4447
+	github.com/onsi/gomega v1.21.1
+	github.com/openshift-kni/cnf-features-deploy v0.0.0-20221007071024-22a40aaee43b
```

Comment:
Why ptp operator has dependency of cnf-features-deploy?

Reply:
fixed, pulled in images.go